### DISCUSSION TITLE
Is the content of the Plutus book worth translation or is it already outdated?

### DISCUSSION THREAD

**Discussion by alcemirsantos** [input-output-hk/plutus#3704] [General]
Hi everyone,

It came to my knowledge existence of the [Plutus: Writing Reliable Smart Contracts](https://leanpub.com/plutus-smart-contracts) book published in 2019 under MIT license by [Lars Brünjes](https://github.com/brunjlar) and Polina Vinogradova.  It seems to me the book once lived in this repo as I could find two related tags ([version 1.0](https://github.com/input-output-hk/plutus/releases/tag/book-v1.0) and its [last update](https://github.com/input-output-hk/plutus/releases/tag/plutus-book-2020)).  My question are: 

1. Is the book still maintained somewhere? If yes, where?
2. If no, is its content still woth a translation or is it already outdated?

**Comment by michaelpj** [comment]
No and no.

**Reply by alcemirsantos** [reply]
is there any other way to learn Plutus, apart the developer portal once the book (and the udemy course might be too) is outdated?
